packages/preview/vlna/0.1.0/vlna.typ

```diff
@@ -0,0 +1,6 @@
+// All characters with less than 3 letters are connected to the next word.
+#show text: body => {
```

Comment:
This should become something like:

```suggestion
#let apply-vlna = body => {
```

(you could also name it `with-vlna` or anything you like, `apply-vlna` is only a suggestion)

And then people can use your package like this:

```typst
#import "@preview/vlna:0.1.0": apply-vlna
#show: apply-vlna

This text will automatically have the "vlna" rule applied to it.
```